### DISCUSSION TITLE
Any issue using EF Core with failsafes + factory timeouts?

### DISCUSSION THREAD

**Discussion by onionhammer** [ZiggyCreatures/FusionCache#289] [Q&A] [answered]
The factory could keep going after the factory timeout has been triggered, doesn't that mean the EF core context could be disposed if it originates outside of the factory; i.e. if the scope is disposed and the context is disposed while the factory is still working

Should EF core contexts be created and disposed within the factory if a failsafe is used?

**Comment by jodydonetti** [comment, marked as answer]
Hi @onionhammer , I think you can take a look at this https://github.com/ZiggyCreatures/FusionCache/discussions/282#discussioncomment-10276690

**Reply by onionhammer** [reply]
Thanks for the reply - Is this documented anywhere? Maybe a 'warning' box is appropriate for this feature?

I also assume this would affect the eager refresh feature

**Reply by jodydonetti** [reply]
> Thanks for the reply - Is this documented anywhere? Maybe a 'warning' box is appropriate for this feature?

Yep, it has been some time I've been thinking where to put that, but I think now I know.
Will update you on this.

> I also assume this would affect the eager refresh feature

Yes, it's in the linked comment 😉
